frontend/src/screens/balance.vue

```diff
@@ -137,6 +145,8 @@ export default {
           backgroundClass: " bg-warning",
           message: "notEnoughInfoForBalance"
         };
+        this.existsCurrentPeriod = period.length !== 0 && period !== undefined;
+        this.currentPeriodHasActions = actions && actions.length !== 0;
```

Comment:
No es necesario comparar contra el 0.
```suggestion
        this.currentPeriodHasActions = actions && actions.length;
```

Reply:
en el caso del periodo está el tema que a veces devuelve un array vacio y otras un objeto, hay que hacer un chequeo fulero

Reply:
aaa cierto, bueno en esos casos no le des importancia a mi comentario entonces :)